includes/language/ukrainian.php

```diff
@@ -41,7 +43,7 @@
     'email_ga_subject' => 'Ваш Google Authenticator флеш код для Teampass',
     'email_ga_text' => 'Вітаю, <a href=\'#link#\'>перейдіть за цим посиланням</a> і зіскануйте QR код за допомогою застосунка Google Authenticator. Це потрибно для налаштування вашого флеш-коду',
     'settings_attachments_encryption' => 'Ввімкнути шифрування для вкладень',
-    'settings_attachments_encryption_tip' => 'ЦЕ НАЛАШТУВАННЯ МОЖЕ ПОШКОДИТИ ІСНУЮЧІ ВКЛАДЕНН�',
+    'settings_attachments_encryption_tip' => 'Ð¦Ð• ÐÐÐ›ÐÐ¨Ð¢Ð£Ð’ÐÐÐÐ¯ ÐœÐžÐ–Ð• ÐŸÐžÐ¨ÐšÐžÐ”Ð˜Ð¢Ð˜ Ð†Ð¡ÐÐ£Ð®Ð§Ð† Ð’ÐšÐ›ÐÐ”Ð•ÐÐÐ',
```

Comment:
As best as I can tell, this Unicode value was not changed by my code, but still shows as different, I have updated the value to the exact same string in GitHub directly, however, this difference still shows.